src/containers/SummaryContainer/index.tsx

```diff
@@ -39,7 +47,12 @@ const SummaryContainer = () => {
         <p>100 XLM</p>
       </CCard>
 
-      <CButton kind="form" content="Create Stream" logo={fluxityLogo} />
+      <CButton
+        kind="form"
+        content="Create Stream"
+        logo={fluxityLogo}
+        onClick={handleClick}
+      />
```

Comment:
summary is a part of the create stream page itself. No need to use this here. Revert the file back to its content